test/buffer_manager_tests.cc

```diff
@@ -1098,6 +1098,14 @@ namespace detail {
 		}
 	}
 
+	TEST_CASE_METHOD(test_utils::runtime_fixture, "set buffer debug name", "[buffer_debug_name]") {
```

Comment:
Maybe also add a test that checks that the name shows up in the generated graph.